### PR TITLE
Add nodeSelector to helm chart

### PR DESCRIPTION
**Pull request by stevededalus**
Addresses #1899 for the dask-distributed charts, adding a nodeSelector option to the configs.

Not sure if this is the right place to be sending a PR, would love some input or pointers.

@bnaul

### CODE REVIEW

File: dask/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: dask
-version: 0.17.1
+version: 0.17.2
```
Comment (bnaul) [inline dask/Chart.yaml]: `0.17.1` is the latest dask version so this probably should remain fixed. Do helm chart versions actually mean anything or does the latest always get pulled...?

**Comment by mrocklin** [comment]
Thanks for the fix!  Can you link to the underlying issue?  Which repository is this from?  I'm curious to see the context of this.

File: dask/Chart.yaml (hunk 1 above)
Comment (mrocklin) [reply dask/Chart.yaml]: Apparently there is a separate appVersion that we'll eventually start using.  I plan to bump this up to 1.0.0 once we get the chart into stable.

**Comment by stevededalus** [comment]
Sorry about that, copy past mistake on the issue link:
https://github.com/kubernetes/charts/issues/1899

Basically just adds a way to allow user to allocate dask workers to different node pools.

**Comment by mrocklin** [comment]
Thanks!  Merging.

**Pull request merged**